recipes/horton/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "horton" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name | lower }} 
+  version: {{ version }}
+
+source:
+  url: https://github.com/theochem/horton/releases/download/{{ version }}/horton-{{ version }}.tar.gz
+  sha256: 5cd76d53368995dd2fba20903a3bddda1161ffa6ffe9f9eef94189ed0a67cc27
+
+build:
+  number: 0
+  skip: true  # [win]
```

```suggestion
  skip: true  # [win]
  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
```

I wonder why we are not using pip install here, though. Any reason?